src/tools/dev/masonry/masonry.py

```diff
@@ -694,7 +694,7 @@ def view_log(fname):
     try:
         child = subprocess.Popen([sys.executable, 
                                   '-m',
-                                  'SimpleHTTPServer',
+                                  'http.server',
```

Comment:
This Popen call doesn't require universal_newlines ?

Reply:
I didn't add the universal_newlines, since no attempt is made to read the output, in which case this isn't necessary.

Reply:
Good to know, thanks for the info.